client/v2/autocli/app.go

```diff
@@ -9,20 +10,23 @@ import (
 	"cosmossdk.io/client/v2/autocli/flag"
 	"cosmossdk.io/core/appmodule"
 	"cosmossdk.io/depinject"
+	"cosmossdk.io/log"
+	"cosmossdk.io/x/tx/signing"
 
 	"github.com/cosmos/cosmos-sdk/client"
 	sdkflags "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec/types"
 )
 
-// AppOptions are autocli options for an app. These options can be built via depinject based on an app config. Ex:
+// Options are input options for an autocli enabled app. These options can be built via depinject based on an app config.
 // Ex:
 //
 //	var autoCliOpts autocli.AppOptions
 //	err := depinject.Inject(appConfig, &encodingConfig.InterfaceRegistry, &autoCliOpts)
```

Comment:
we should update this, to show that `ProvideAppOptions` is required in the app config